sources/editor/Xenko.Core.Assets.Editor/View/DataGridEx.cs

```diff
@@ -11,10 +11,16 @@
 using Xenko.Core.Extensions;
 using Xenko.Core.Presentation.Extensions;
 
-using Xceed.Wpf.DataGrid;
-
 namespace Xenko.Core.Assets.Editor.View
 {
+    /// <summary>
+    /// Xenko generic wpf data grid. 
+    /// </summary>
+    public class DataGridEx : DataGrid
```

Comment:
Curious, any reason to not use the `DataGrid`  type directly?

Reply:
I figured in the future we would want to add the replaced functionality, and thought it would be best to have a single data grid which we can improve upon without needing to refactor later.

Reply:
Sounds good, fine with keeping it then!
In that case, could you add such an explanation in the class comment, so that future maintainers understand why it's there?